authors/settings.py

```diff
@@ -143,12 +143,13 @@
     'NON_FIELD_ERRORS_KEY': 'error',
 }
 
-django_heroku.settings(locals())
-
-# Setup EMAIL configs
+#Configurations for the smtp mail server
 EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
 EMAIL_HOST = os.getenv('EMAIL_HOST')
 EMAIL_HOST_USER = os.getenv('EMAIL_HOST_USER')
 EMAIL_HOST_PASSWORD = os.getenv('EMAIL_HOST_PASSWORD')
 EMAIL_PORT = os.getenv('EMAIL_PORT')
-EMAIL_USE_TLS = os.getenv('EMAIL_USE_TLS')
\ No newline at end of file
+EMAIL_USE_TLS = os.getenv('EMAIL_USE_TLS')
+EMAIL_USE_TLS = True
```

Comment:
Consider getting rid of one `EMAIL_USE_TLS` constant. It's is redundant to declare twice

Reply:
Ok. already solved that